src/components/HomePage/RepairServices/RepairServices.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled, { useTheme } from "styled-components";
+import AlignedContainer from "../../UI/AlignedContainer/AlignedContainer";
+import Column from "./Column/Column";
+import SecondaryHeading from "../../UI/Headings/SecondaryHeading";
+import Sentence from "../../UI/Sentence/Sentence";
+import RegularButton from "../../UI/Buttons/RegularButton";
+import ServicesList from "./List/ServicesList";
+import { servicesData1, servicesData2 } from "./data";
+import DarkLayer from "./DarkLayer/DarkLayer";
+
+const Container = styled.section`
+  margin-top: 200px;
+  background-image: url("/images/RepairServices/Repair.png");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+`;
+
+const Content = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const RepairServices = () => {
+  const myTheme = useTheme();
+  return (
```

Comment:
i prefer to use rendering list concept, for more reusability, imagine if you want to modify something in the future it is more readable and more easy to work with